### PR TITLE
Update dashboard to v0225

### PR DESCRIPTION
**Pull request by breeswish**
### What problem does this PR solve? 

Update dashboard to v0225

### What is changed and how it works?

Update dashboard to v0225

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)

### CODE REVIEW

File: pkg/dashboard/apiserver/apiserver.go [hunk 1]
```diff
@@ -23,19 +23,35 @@ import (
...
 	"github.com/pingcap/log"
+	"go.etcd.io/etcd/clientv3"
+	"go.uber.org/zap"
+
 	"github.com/pingcap/pd/v4/pkg/dashboard/keyvisual/input"
 	"github.com/pingcap/pd/v4/server"
-	"go.uber.org/zap"
 )
 
+var _ dashboardpd.EtcdProvider = (*PdEtcdProvider)(nil)
```
Comment (HunDunDM) [inline pkg/dashboard/apiserver/apiserver.go]: What is the use of this line?

File: pkg/dashboard/apiserver/apiserver.go [hunk 2]
```diff
@@ -53,20 +68,28 @@ func NewService(ctx context.Context, srv *server.Server) (http.Handler, server.S
 		panic(err)
 	}
 
+	etcdProvider := &PdEtcdProvider{srv: srv}
+
+	tidbForwarder := tidb.NewForwarder(tidb.NewForwarderConfig(), etcdProvider)
+	// FIXME: Handle open error
+	tidbForwarder.Open()        //nolint:errcheck
+	defer tidbForwarder.Close() //nolint:errcheck
```
Comment (HunDunDM) [inline pkg/dashboard/apiserver/apiserver.go]: Should it move to AddCloseCallback?

File: pkg/dashboard/apiserver/apiserver.go (hunk 1 above)
Comment (breeswish) [reply pkg/dashboard/apiserver/apiserver.go]: Ensure that PdEtcdProvider implements EtcdProvider

**Pull request merged**